Sources/Rendering/Core/Camera/index.js

```diff
@@ -39,6 +39,14 @@ function vtkCamera(publicAPI, model) {
   const newPosition = vec3.create();
   const newFocalPoint = vec3.create();
 
+  // Internal Functions that don't need to be public
+  function computeViewPlaneNormal() {
```

Comment:
Moved to an internal function, off of publicAPI, since any publicAPI function that changes the directionofProjection should call this internally, a consumer of VTK.js should never need to call this.